### ISSUE TITLE
custom_deps 0.3, installing dependent packages does not work

### ISSUE THREAD

**Issue by mories76** [home-assistant/hassio-addons#355]
"pypi": [
    "pyaes",
    "sleekxmpp",
    "nefit-client"

When I try to install these packages. The installation of nefit-client failes because pyaes is not available.
See log below. 

[Info] Remove old deps
[Info] Install pypi modules into deps
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-m7qaix0u/nefit-client/
You are using pip version 9.0.1, however version 10.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
[Error] Can't install pypi packages!
Collecting pyaes
  Downloading https://files.pythonhosted.org/packages/44/66/2c17bae31c906613795711fc78045c285048168919ace2220daa372c7d72/pyaes-1.6.1.tar.gz
Collecting sleekxmpp
  Downloading https://files.pythonhosted.org/packages/13/ca/2a73521df7696b4cf9a9820cd61097c1c88cbb9da67a712146e622426ca1/sleekxmpp-1.3.3.tar.gz (845kB)
Collecting nefit-client
  Downloading https://files.pythonhosted.org/packages/c8/74/57b04fc5ec52cec473ebc13cd63852f26fd98ec3a9181befbbc59438dc67/nefit-client-0.2.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-m7qaix0u/nefit-client/setup.py", line 5, in <module>
        from nefit import version
      File "/tmp/pip-build-m7qaix0u/nefit-client/nefit/__init__.py", line 7, in <module>
        from pyaes import AESModeOfOperationECB, Encrypter, Decrypter, PADDING_NONE
    ModuleNotFoundError: No module named 'pyaes'
    
    ----------------------------------------

**Comment by mories76** [comment]
@pvizeli @georgemarshall 
I just noticed there was an update to the custom_deps addon. But I am not sure what has been fixed.
I cannot install two packages at the same time where the second one is depending on the first one to be installed. I really do think a solution would be to loop through the array and do multiple pip install command, as proposed in PR #346 

OFFTOPIC
I also think that Nefit Easy for which we need these extra dependencies, should have a proper component. But I lack the skill to build a good component.
